.github/workflows/Documentation.yaml

```diff
@@ -25,11 +27,16 @@ on:
         description: 'Commit SHA containing the updated _releases.json'
         required: true
         type: string
+      is-workflow_call:
```

Comment:
you don't need this. just use `${{ github.event_name }}`

Reply:
Yes that's true because it will depend on where it's being called.

But that highlights exactly the issue you have here:
 - sometimes, even if being used as a workflow_call, you may want to run the validation check (e.g. the `Documentation` workflow is called from a remotely dispatched workflow) (in which case it is good that `event_name` is `workflow_dispatch`

I know - "the dispatched workflow already does the validation". But that's not future proof and creates a loose dependency that can very easily break.

We need a more robust condition to decide when to make the check.
I'm thinking about using a combo of `github.repository` (we only care about validation when running in the `oci-factory` repo - others who _call this from outside don't need this validation) with `github.event` (the name is still useful, i.e. if it says `workflow_dispatch`, then you know you want to run it, even if this workflow was called by another) with `inputs` (your approach is good but introduces an unnecessary variable - just use another that is unique to the workflow_call)